appcues/src/main/java/com/appcues/trait/appcues/TargetElementTrait.kt

```diff
@@ -63,10 +67,16 @@ internal class TargetElementTrait(
 
         val weightedViews = strategy.findMatches(selector)
             // if the result is null (not just empty) - that means the UI layout was not available at all
-            ?: throw AppcuesTraitException("could not read application layout information")
+            ?: throw AppcuesTraitException(
+                message = "could not read application layout information",
+                retryMilliseconds = retryMilliseconds,
+            )
 
         if (weightedViews.isEmpty()) {
-            throw AppcuesTraitException("no view matching selector ${selector.toMap()}")
+            throw AppcuesTraitException(
+                message = "no view matching selector ${selector.toMap()}",
+                retryMilliseconds = retryMilliseconds,
+            )
```

Comment:
this is the typical selector not found - but the other two items in this class are also considered possibly recoverable as a view loads or animates in.